From ba61fe50bb378e79f2b5c92d63df52d54104383e Mon Sep 17 00:00:00 2001
From: tstegema <tstegema@90b65887-3827-0410-9a23-83215b262276>
Date: Thu, 12 Jun 2014 18:11:02 +0000
Subject: [PATCH 0550/1076] MANTIS 0019643 extension
 Platform_Extension_CUSS_1_2: Single-application mode device support

platform.idl
V1.8
added to ComponentIF for MediaOffRoller
code beginMediaRecord(in string contextName);
code resetMediaRecord();


git-svn-id: svn://localhost/SelfServiceCommon/trunk@687 90b65887-3827-0410-9a23-83215b262276
---
 SelfServiceCommon/Massai/idl/Massai/platform.idl | 15 +++++++++++++--
 1 file changed, 13 insertions(+), 2 deletions(-)

diff --git a/SelfServiceCommon/Massai/idl/Massai/platform.idl b/SelfServiceCommon/Massai/idl/Massai/platform.idl
index a7be53a1..0b86260c 100644
--- a/SelfServiceCommon/Massai/idl/Massai/platform.idl
+++ b/SelfServiceCommon/Massai/idl/Massai/platform.idl
@@ -27,7 +27,7 @@
 
 module PlatformManagement
 {
-    const string interface_version = "1.7";
+    const string interface_version = "1.8";
 
     /** Data type for text lists. */
     typedef sequence<string> list;
@@ -133,6 +133,16 @@ module PlatformManagement
         /** Tells the device component to terminate.
             @return RC_OK. */
         code terminate();
+
+
+        /** prepare a device for using by CLA in media-roll-off mode. 
+            record events for playback after next prepareDevice
+        */
+        code beginMediaRecord(in string contextName);
+        
+        /** delete events, recorded for media-roll-off mode
+        */
+        code resetMediaRecord();
     };
 
     enum DataStatusExtension
@@ -207,7 +217,8 @@ module PlatformManagement
     interface PlatformIF
     {
 
-    typedef sequence<LoggingComponentData> LoggingComponentDataList;   
+        typedef sequence<LoggingComponentData> LoggingComponentDataList;   
+        
         void initLogging(in string realComponentName, in LoggingComponentDataList init);
         void logCounter(in LoggingCounterData lcd);
         void logCounterFailure(in string unitId, in LoggingClass unitClass, in long numberOfFailed);
-- 
2.41.0.windows.1

